backend/src/main/java/com/huellapositiva/application/exception/SomeFieldIsNullException.java

```diff
@@ -0,0 +1,11 @@
+package com.huellapositiva.application.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
When the validation of the request body fails the return code should be 400

Reply:
Ok, is done.